tests/dialects/test_dialect.py

```diff
@@ -4303,3 +4304,147 @@ def test_subquery_unwrap(self):
             "WITH sub_query AS (SELECT a FROM table) ((((SELECT a FROM sub_query))))",
             "WITH sub_query AS (SELECT a FROM table) SELECT a FROM sub_query",
         )
+
+    def test_initcap(self):
```

Comment:
Just double-checking: the results have been checked in source / target engines & they match, right?